app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,10 @@
+class PostsController < ApplicationController
+  before_action :authenticate_user!
+  def index; end
```

Comment:
- [ ] We should have the posts displaying. [Requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone3/posts2.png).

- [ ] **You should have a controller for the users**, and it should have the index action filled as well. [Requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone3/posts2.png).